transcom/mymove
# INT B-21376 I-13558

Downstream of https://github.com/transcom/mymove/pull/14253 - don't merge until that is merged into INT

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21376)
## [Issue ticket](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A1034466)

## Summary

They wanted to see the UB allowance up top where the weight allowance is shown, so Imma show it.

### How to test

1. Set up your env so you can create UB shipments
2. After adding your orders information, go back to your home page and check out your headers
3. Should see a `UB allowance` up there lookin' all pretty

## Screenshots
![Screenshot 2024-11-21 at 9 40 51 AM](https://github.com/user-attachments/assets/8cb9383a-6705-4101-a3b6-e56a2d75c436)

![Screenshot 2024-11-21 at 12 32 37 PM](https://github.com/user-attachments/assets/b7c55191-a23b-4f17-bc34-57e36441c5f0)

![Screenshot 2024-11-21 at 12 33 26 PM](https://github.com/user-attachments/assets/8b22b2cd-9f47-4c44-979e-2251fa73b429)
